pkgs/applications/audio/luppp/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchFromGitHub
+, meson
+, ninja
+, pkgconfig
+, jack2
+, cairo
+, liblo
+, libsndfile
+, libsamplerate
+, ntk
+}:
+
+stdenv.mkDerivation rec {
+  name = "luppp-${version}";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "openAVproductions";
+    repo = "openAV-Luppp";
+    rev = "release-${version}";
+    sha256 = "194yq0lqc2psq9vyxmzif40ccawcvd9jndcn18mkz4f8h5w5rc1a";
+  };
+
+  buildInputs = [
+    meson ninja pkgconfig jack2 cairo liblo libsndfile libsamplerate ntk
+  ];
+
+  buildPhase = ''
```

Comment:
No need to explicitly add `buildPhase`, it will be automatically created by ninja setup-hook.